server/src/lib.rs

```diff
@@ -24,5 +24,5 @@ pub async fn grpc_fallback(uri: Uri) -> impl IntoResponse {
         tonic::Code::NotFound,
         format!("{} is not a supported gRPC method", uri.path()),
     )
-    .into_http()
+    .into_http::<axum::body::Body>()
```

Comment:
@Tpt not fully confident on this one.
TL;DR: now the underlying type has a `<D : Default>` generic attached to it, and the "old" type was a private one (`tonic::body::BoxBody`)

Reply:
I think it's fine. The return type is `IntoResponse` so the body will be converted into a `axum::body::Body` anyway. So, better to directly return this type.